components/MyPlants.js

```diff
@@ -22,13 +22,24 @@ const StyledImage = styled(Image)`
   height: 20px;
 `;
 
+const StyledButton = styled.button`
+  background: none;
+  cursor: pointer;
+  padding: 5px;
+  margin-top: 10px;
+`;
+
 const BackButton = styled.button`
   background: none;
   cursor: pointer;
 `;
 
-export default function MyPlants({ plants, handleToggleMyPlants }) {
-  const myPlants = plants.filter((info) => info.isMyPlant);
+export default function MyPlants({
+  plants,
+  isMyPlantFunction,
+  handleToggleMyPlants,
+}) {
+  const myPlants = plants.filter((plant) => isMyPlantFunction(plant.id));
```

Comment:
Since the function is a function, it isn't needed to implement the word "function" in the function-naming: "isMyPlant" will do it.

Reply:
Thanks Steffen.
We just want to differentiate between our key "isMyPlant" and "isMyPlantFunction".